e2e_test/source_inline/pubsub/pubsub.slt

```diff
@@ -1,15 +1,20 @@
+control substitution on
+
+system ok
+e2e_test/source_inline/pubsub/prepare-data.rs
```

Comment:
1. Is there any documentation on how to use `system ok`?
2. It does not look like really *inline* to me ... Just replacing shell script with rust